Community_Lighting_MZ.js

```diff
@@ -8,12 +8,12 @@ var Community = Community || {};
 Community.Lighting = Community.Lighting || {};
 Community.Lighting.name = "Community_Lighting_MZ";
 Community.Lighting.parameters = PluginManager.parameters(Community.Lighting.name);
-Community.Lighting.version = 4.2;
+Community.Lighting.version = 4.3;
 var Imported = Imported || {};
 Imported[Community.Lighting.name] = true;
 /*:
 * @target MZ
-* @plugindesc v4.2 Creates an extra layer that darkens a map and adds lightsources! Released under the MIT license!
+* @plugindesc v4.3 Creates an extra layer that darkens a map and adds lightsources! Released under the MIT license!

```

Comment:
We should have a version history markdown doc. Maybe I'll make one later.

Reply:
Version 4.3 is where my map notes were added. And the softer lights option I don't know about the rest though.